### PR TITLE
Ch1470/none blocking add topic to capability

### PR DESCRIPTION
**Pull request by kim-lindhard-dfds**
First time in a while I have done a fire and forget thread

### CODE REVIEW

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |3|
| **Quality Gates** |OK|
| **Description** |The change is low risk as it touches less modules than your typical change set. It adds 65 lines, deletes 4 lines of code|
| **Warnings** |Delta Analysis Info<br/><ul><li>1 files in this change set did not exist with the same name on the main branch. This is probably because those files have been renamemd. Skipping code health checks for those files.</li></ul>|

File: src/CapabilityService.Tests/Scenarios/AddTopicToCapability.cs [hunk 1]
```diff
@@ -64,21 +66,48 @@ private async Task When_a_topic_is_added()
...
+		public async Task<TResult> DoUntilResultOr5Sec<TResult>(Func<Task<TResult>> function)
+		{
+			var endTime = DateTime.Now.AddSeconds(5);
+			dynamic result = null;
+			do
+			{
+				result = await function();
+				if (result == null)
+				{
+					Thread.Sleep(1000);
```
Comment (SEQUOIIA) [inline src/CapabilityService.Tests/Scenarios/AddTopicToCapability.cs:106]: :eyes:

File: src/CapabilityService.WebApi/Features/Kafka/Infrastructure/Api/TopicController.cs [hunk 2]
```diff
@@ -80,14 +81,22 @@ public async Task<IActionResult> AddTopicToCapability(string id, [FromBody] Topi
 
 
 				await _topicDomainService.CreateTopic(
-					topic,
-					input.DryRun
+					topic: topic,
+					dryRun: true
```
Comment (SEQUOIIA) [inline src/CapabilityService.WebApi/Features/Kafka/Infrastructure/Api/TopicController.cs]: Wait, why?

**Review by SEQUOIIA — APPROVED** [review]
Assuming it passes tests, looks fine to me.

**Pull request merged**